shell/src/main/java/alluxio/shell/command/CopyFromLocalCommand.java

```diff
@@ -238,6 +243,11 @@ private void copyPath(File src, AlluxioURI dstPath) throws AlluxioException, IOE
       mFileSystem.createDirectory(dstPath);
       List<String> errorMessages = new ArrayList<>();
       File[] fileList = src.listFiles();
+      if (fileList == null) {
+        String errMsg = String.format("copyFromLocal %s %s partially failed!",
```

Comment:
Similar comment to above, it's probably more actionable for the user if we tell them the directory where we couldn't list the files, and we should add the message to `errorMessages` so that the destination directory gets cleaned up.